src/merkle/merkle-objects.ts

```diff
@@ -0,0 +1,98 @@
+import CID from "cids";
+
+import { Doctype } from '@ceramicnetwork/common';
+import {
+  CompareFunction,
+  MergeFunction, MetadataFunction,
+  Node,
+  TreeMetadata,
+} from './merkle';
+
+import { logger } from '../logger';
+import { IpfsService } from '../services/ipfs-service';
+
+import { BloomFilter } from 'bloom-filters';
+
+const BLOOM_FILTER_TYPE = "jsnpm_bloom-filters";
+const BLOOM_FILTER_FALSE_POSITIVE_RATE = 0.0001
```

Comment:
Just FYI:
```
# node
Welcome to Node.js v14.15.5.
Type ".help" for more information.
> const { BloomFilter } = require('bloom-filters')
undefined
> const NUM_ITEMS = 10000
undefined
> const FALSE_POSITIVE_RATE = 0.0001
undefined
> BloomFilter.create(NUM_ITEMS, FALSE_POSITIVE_RATE).size
191702

```

Reply:
Isn't this an extremely low false positive rate?

Reply:
Well, the only cost to a lower false positive rate is a greater size of the resulting filter.  From my simple experiments, it seemed that the number of entries had a bigger impact on the filter size than the false positive rate:

```
> BloomFilter.create(100, 0.0001).size
1918
> BloomFilter.create(1000, 0.0001).size
19171
> BloomFilter.create(10000, 0.0001).size
191702
> BloomFilter.create(100000, 0.0001).size
1917012
> BloomFilter.create(10000, 0.01).size
95851
> BloomFilter.create(10000, 0.001).size
143776
> BloomFilter.create(10000, 0.0001).size
191702
> BloomFilter.create(10000, 0.00001).size
239627
```

Reply:
We could go down to `.001`, but it looks like a 10x increase in false positives only gains us ~25% space usage improvement

Reply:
Ok, thanks for the rundown. I'm not sure what the best call is here actually. I guess we can start with your configuration and then see if any issues arise on the indexing side.

Reply:
Care to approve the PR?